test/gradcheck.jl

```diff
@@ -1707,3 +1708,17 @@ end
     @test s == 0.0
     @test gs == (nothing,)
 end
+
+@testset "Fix1 and Fix2" begin
+    @test gradcheck(x -> prod(Base.Fix1(+, 1), x), randn(100))
+    @test gradcheck(x -> prod(Base.Fix2(+, 1), x), randn(100))
+
+    # check the execution times compared with a closure
+    # https://github.com/FluxML/Zygote.jl/issues/957
+    x = randn(100)
+    tclosure = @belapsed(gradient($(x -> prod(y -> y + 1, x)), $x))
+    tfix1 = @belapsed(gradient($(x -> prod(Base.Fix1(+, 1), x)), $x))
+    tfix2 = @belapsed(gradient($(x -> prod(Base.Fix2(+, 1), x)), $x))
+    @test tfix1 < 2 * tclosure
+    @test tfix2 < 2 * tclosure
```

Comment:
```suggestion
```
We can't really add these benchmarks to tests in a systematic way, even when fixing major performance problems, long testing time makes development very annoying. I suggest we just leave these benchmarks out.

Reply:
It would not be crazy to start a collection of benchmarks somewhere, perhaps to be run manually before tagging?